source/imaer-gml/src/main/java/nl/overheid/aerius/gml/GMLWriter.java

```diff
@@ -253,6 +249,27 @@ public File writeArchiveContributionsToFile(final File dir, final List<Calculati
     return file.toFile();
   }
 
+  /**
+   * Build GML and write it directly to file.
+   * The file will be generated in the supplied directory.
+   *
+   * @param dir The directory to write the file to.
+   * @param scenario Scenario containing all scenario related data
+   * @param metaData The metadata to write for this file.
+   * @param fileId The ID that should be reflected in the filename.
+   * @param fileNamePart The optional filename that will be incorporated in the resulting file's name.
+   * @return The file generated.
+   * @throws AeriusException When exception occurred generating the GML.
+   * @deprecated Use {@link #writeToFile(Path, IsScenario, MetaDataInput)}
```

Comment:
Do you mean the function above? If so, the variable names don't match

Reply:
Yes refers to new method, the reference states the types of the method not the variable names.

Reply:
Ah, check 😅